src/backer/cardaning.py

```diff
@@ -69,44 +70,75 @@ def __init__(self, hab, ks=None):
     def updateTip(self, tipHeight):
         self.tipHeight = tipHeight
 
-    def publishEvent(self, event: bytes):
-        self.pending_kel = event + self.pending_kel
-
-    def submitKelTx(self, kel):
-        try:
-            if kel:
-                # Build transaction
-                builder = pycardano.TransactionBuilder(self.context)
-                builder.add_input_address(self.funding_addr)
-                builder.add_output(pycardano.TransactionOutput(self.funding_addr, pycardano.Value.from_primitive([TRANSACTION_AMOUNT])))
-                kel_data = bytes(kel)
-                # Chunk size
-                # bytearrays is not accept
-                value = [kel_data[i:i + 64] for i in range(0, len(kel_data), 64)]
-
-                # Metadata. accept int key type
-                builder.auxiliary_data = pycardano.AuxiliaryData(pycardano.Metadata({1: value}))
-
+    def addToQueue(self, event):
+        serder = serdering.SerderKERI(raw=event)
+        self.keldb_queued.pin(keys=(serder.pre, serder.said), val=serder)
+
+    def removeFromQueue(self, event):
+        serder = serdering.SerderKERI(raw=event)
```

Comment:
Why do we need to convert here? Doesn't event contain the pre and said directly?

Reply:
The event contains the pre and said directly, but when we get the event from database, it still is bytes or str, so we need to convert to serder object to get pre and said